server/src/main/java/io/crate/metadata/GeoReference.java

```diff
@@ -49,11 +50,12 @@ public class GeoReference extends SimpleReference {
 
     public GeoReference(int position,
                         ReferenceIdent ident,
+                        boolean nullable,
                         @Nullable String tree,
                         @Nullable String precision,
                         @Nullable Integer treeLevels,
                         @Nullable Double distanceErrorPct) {
-        super(ident, RowGranularity.DOC, DataTypes.GEO_SHAPE, position, null);
+        super(ident, RowGranularity.DOC, DataTypes.GEO_SHAPE, ColumnPolicy.DYNAMIC, IndexType.PLAIN, nullable, false, position, null);
```

Comment:
doc values is always false and not taken from what we [have](https://github.com/crate/crate/blob/master/server/src/main/java/io/crate/metadata/doc/DocIndexMetadata.java#L449) in `DocIndexMetadata` but I think it's fine since we don't have them for GEO SHAPE anyway.